benchmark/coverage_analyzer.py

```
@@ -0,0 +1,1134 @@
+        
+        return '\n'.join(lines)
+    
+    def _is_mutation_killed(self, source_file: str, mutated_content: str, test_files: List[str]) -> bool:
+        """Test if a mutation is killed by tests"""
+        try:
+            # Write mutated file
+            with open(source_file, 'w') as f:
+                f.write(mutated_content)
+            
+            # Run tests
+            result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)